contrib/format-maprdb/src/main/java/org/apache/drill/exec/store/mapr/db/MapRDBSubScan.java

```diff
@@ -57,34 +59,37 @@ public MapRDBSubScan(@JacksonInject StoragePluginRegistry engineRegistry,
                        @JsonProperty("regionScanSpecList") List<MapRDBSubScanSpec> regionScanSpecList,
                        @JsonProperty("columns") List<SchemaPath> columns,
                        @JsonProperty("maxRecordsToRead") int maxRecordsToRead,
-                       @JsonProperty("tableType") String tableType) throws ExecutionSetupException {
+                       @JsonProperty("tableType") String tableType,
+                       @JsonProperty("schema") TupleSchema schema) throws ExecutionSetupException {
```

Comment:
Ideally we should use `TupleMetadata` instance not conrete implementation, DRILL-7272 will add such support, could you please add todo with Jira number in all places where `TupleSchema` is used so it could be easily tracked.